snap7/util.py

```diff
@@ -451,7 +451,7 @@ def set_string(bytearray_: bytearray, byte_index: int, value: str, max_size: int
         bytearray_[byte_index + 2 + r] = ord(' ')
 
 
-def get_string(bytearray_: bytearray, byte_index: int, max_size: int) -> str:
```

Comment:
this breaks backwards compatibility, right?

Reply:
`max_size` is important for set/write methods, where you also have to be aware of the max. length.

If you get an already readed bytearray, the information is already in the bytearray.